casepro/cases/forms.py

```diff
@@ -18,8 +17,9 @@ def __init__(self, *args, **kwargs):
         org = kwargs.pop('org')
         super(PartnerForm, self).__init__(*args, **kwargs)
 
+        self.fields['primary_contact'].queryset = org.get_users()
```

Comment:
@rowanseymour If that is the case, I think we should remove the `primary_contact` field from the partner `create` form. If you agree with that, are you happy if we use two different forms, like `PartnerCreateForm` and `PartnerUpdateForm`?